activerecord/lib/active_record/inheritance.rb

```diff
@@ -49,8 +49,18 @@ def new(*args, &block)
         if abstract_class? || self == Base
           raise NotImplementedError, "#{self} is an abstract class and cannot be instantiated."
         end
+        
```

Comment:
I need a help as when I did `relation.scoping{}` it was not fullfil the test can any one help me?